README.md

```diff
@@ -19,14 +19,17 @@ Neural network-based singing voice synthesis library for research.
 ## Installation
 
 - Python 3.6 or newer
-- [nnmnkwii](https://github.com/r9y9/nnmnkwii): **development** version (master branch) is required
-- [pysinsy](https://github.com/r9y9/pysinsy) **development** version is required. Please have a look at the repostiory for installation.
 - Pytorch >= 1.x
 
 Note that packages listed above should be manually installed. After installing them, you can run:
 
 ```
-python setup.py develop
+git submodule init
+git submodule update
+cd 3rdparty
+./install.sh
+cd ..
+python setup.py
```

Comment:
I am not sure what exactly happens if we omit `install` or `develop`. Would it be okay to have `python setup.py develop` as before?